lib/components/ColorPalette.js

```diff
@@ -1,20 +1,10 @@
 import React from "react";
 import classnames from "classnames";
 
-const COLORS = [
-  { from: "purple-1000", to: "purple-1050", hexCode: "#6671E5" },
-  { from: "ash-1000", to: "ash-1050", hexCode: "#606C88" },
-  { from: "kimoby-1000", to: "kimoby-1050", hexCode: "#396AFC" },
-  { from: "turquoise-1000", to: "turquoise-1050", hexCode: "#136A8A" },
-  { from: "shrimpy-1000", to: "shrimpy-1050", hexCode: "#E43A15" },
-  { from: "veryblue-1000", to: "veryblue-1050", hexCode: "#0575E6" },
-  { from: "fbmessenger-1000", to: "fbmessenger-1050", hexCode: "#00C6FF" }
-];
-
-const ColorPalette = ({ color, onChange }) => {
+const ColorPalette = ({ color, colorList, onChange }) => {
```

Comment:
@goutham-subramanyam _a Why not pass Colors as the default value so that we can keep it backward compactable.

```suggestion
const COLORS = [
  { from: "purple-1000", to: "purple-1050", hexCode: "#6671E5" },
  { from: "ash-1000", to: "ash-1050", hexCode: "#606C88" },
  { from: "kimoby-1000", to: "kimoby-1050", hexCode: "#396AFC" },
  { from: "turquoise-1000", to: "turquoise-1050", hexCode: "#136A8A" },
  { from: "shrimpy-1000", to: "shrimpy-1050", hexCode: "#E43A15" },
  { from: "veryblue-1000", to: "veryblue-1050", hexCode: "#0575E6" },
  { from: "fbmessenger-1000", to: "fbmessenger-1050", hexCode: "#00C6FF" }
];

const ColorPalette = ({ color, colorList = COLORS, onChange }) => {
```

Reply:
1. The colors will get purged and we'll need to add them to safelist
2. The colors are mostly product dependent. There is no color reuse. So no use in keeping default colors.

@edwinbbu _a